src/Maker/MakeRegistrationForm.php

```diff
@@ -298,7 +298,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 UniqueEntity::class,
                 [
                     'fields' => [$usernameField],
-                    'message' => sprintf('There is already an account with this '.$usernameField),
+                    'message' => sprintf('There is already an account with this %s', $usernameField),
```

Comment:
The message sounds weird

Reply:
you not wrong. it made sense at the time...